HowTos/SAML_Integration_PingOne_IdP.rst

```diff
@@ -0,0 +1,259 @@
+
+Continue with updating Aviatrix SAML Endpoint by visiting one of the following links based on your use case:
+
+#. If integrating PineOne IdP with `Controller Login SAML Config <https://docs.aviatrix.com/HowTos/Controller_Login_SAML_Config.html#config-34>`_
+
+#. If integrating PineOne IdP with `OpenVPN with SAML Authentication <https://docs.aviatrix.com/HowTos/VPN_SAML.html#config-34>`_
+
+   +-------------------------+-------------------------------------------------+
+   | Field                   | Value                                           |
+   +=========================+=================================================+
+   | Endpoint Name           | ``[Endpoint Name]`` (Use the same name you      |
+   |                         | entered in the PingONe Application previously)  |
+   +-------------------------+-------------------------------------------------+
+   | IdP Metadata Type       | URL                                             |
+   +-------------------------+-------------------------------------------------+
+   | IdP Metadata URL        | ``URL copied from PingOne`` (IdP metadata URL)  |
+   +-------------------------+-------------------------------------------------+
+   | Entity ID               | Select `Hostname`                               |
+   +-------------------------+-------------------------------------------------+
+   | Custom SAML Request     | Check the box and either copy the below format  |
+   | Template                | into the prompt text box or modify it           |
+   +-------------------------+-------------------------------------------------+
+
+   |pingone_idp_reformat_custom_saml_request_template|
+   
+.. code-block:: XML
+   <?xml version="1.0" encoding="UTF-8"?>
+   <samlp:AuthnRequest xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" ID="$ID" Version="2.0" IssueInstant="$Time" Destination="$Dest" ForceAuthn="false" IsPassive="false" ProtocolBinding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" AssertionConsumerServiceURL="$ACS">
+      <saml:Issuer xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion">$Issuer</saml:Issuer>
+      <samlp:NameIDPolicy xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" Format="urn:oasis:names:tc:SAML:2.0:nameid-format:transient" SPNameQualifier="$SPNameQualifier" AllowCreate="true"></samlp:NameIDPolicy>
+      <samlp:RequestedAuthnContext xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" Comparison="exact"><saml:AuthnContextClassRef xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion">urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport</saml:AuthnContextClassRef>
+      </samlp:RequestedAuthnContext>
+   </samlp:AuthnRequest>
+   
+.. _pingone_test_integration:
+
+Step 5. Test the Integration
+#############################
+
```

Comment:
In https://docs.aviatrix.com/HowTos/SAML_Integration_Okta_IdP.html, There is a tip (that tells the user to assign users to the new application).

Is it intentional here that we don't mention this step anymore?

Reply:
PingOne for customer does not have that concept